### PR TITLE
Revert last changes + make header like the docs

### PR DESCRIPTION
**Pull request by diversemix**

### CODE REVIEW

File: src/core/utils/createApolloClient.ts [hunk 1]
```diff
@@ -11,14 +11,15 @@ import { createUploadLink } from 'apollo-upload-client';
 export default (host: string): ApolloClient<unknown> => {
     const apiLink = createUploadLink({
         uri: `${host}/graphql`, // use https for secure endpoint,
+        credentials: 'include',
```
Comment (diversemix) [inline src/core/utils/createApolloClient.ts:14]: I think this might be the key

**Comment by diversemix** [comment]
Do we need the api to be `fetch` compatible /cc @will-byrne / @Cooryd ?

**Comment by Cooryd** [comment]
I believe `fetch` is the browsers default http request api so it should be as thats likely what's being used under the hood by apollo-client (don't quote me on that)

**Comment by diversemix** [comment]
I'll leave it like this for now ... but if you look here - there are other options we may need: https://www.apollographql.com/docs/link/links/http/#options

/cc @Cooryd @will-byrne

**Comment by Cooryd** [comment]
> I'll leave it like this for now ... but if you look here - there are other options we may need: https://www.apollographql.com/docs/link/links/http/#options
> 
> /cc @Cooryd @will-byrne

This should all be fine as we're targeting modern browsers (ie11+) which all have fetch

**Pull request merged**